src/utils.ts

```diff
@@ -36,6 +38,28 @@ export const hasAccounts = async (web3: Web3) => {
   return accounts.length > 0;
 };
 
+/**
+ * Validates the given address
+ *
+ * @param address
+ *
+ * @returns
+ * true if valid, false if not
+ */
+export const isValidAddress = (address: string) => rskjutils.isValidAddress(address);
+
+/**
+ * Validates the given checksum address for RSK
+ *
+ * @param address
+ *
+ * @returns
+ * true if valid, false if not
+ */
+export const isValidChecksumAddress = (
+  address: string,
+) => rskjutils.isValidChecksumAddress(address, ChainId.RSK_MAINNET);
```

Comment:
If it is composed with testnet? The lib should factory its own checksum address encoder.